### DISCUSSION TITLE
what is the best way to update as of this moment?

### DISCUSSION THREAD

**Discussion by ILudw1g** [pivpn/pivpn#1869] [Q&A]
pivpin -up returns:

^[[6~^[[6~[2024-10-19T16:19:04+0100]: ::: The updating functionality for PiVPN scripts is temporarily disabled
[2024-10-19T16:19:04+0100]: ::: To keep the VPN (and the system) up to date, use:
[2024-10-19T16:19:04+0100]:         apt update; apt upgrade

maybe method 1 installation will update it?:
curl -L https://install.pivpn.io | bash

**Comment by orazioedoardo** [comment]
It depends on what you mean of updating. If you have a very old install lacking current feature you will need to reinstall. If you just want to keep openvpn and wireguard up-to-date, use the package manager as the output suggests.

**Comment by ILudw1g** [comment]
thanks for your response, 

by updating i mean the latest version of pivpn that is available? 
what is the command to check the version, maybe its up to date already?

to update with package manager is it:
sudo apt update pivpn

because its not returning anything for me for some reason

sudo apt update wireguard 
also not returning anything for some reason

**Reply by orazioedoardo** [reply]
Run `pivpn -d`, will tell you the commit and date of installation at the beginning.

**Comment by ILudw1g** [comment]
pivpn -d
::: Generating Debug Output
/etc/pivpn/wireguard/setupVars.conf: line 8: 192.168.0.50/24: No such file or directory
::::            PiVPN debug              ::::
=============================================
::::            Latest commit            ::::
Branch: master
Commit: 20d3a4ccd4f2d080d6f9a13a74e6369aa9d54476
Author: Orazio
Date: Mon Jan 29 08:53:41 2024 +0100
Summary: fix(install): don't treat http errors as unavailable ipv6

**Reply by orazioedoardo** [reply]
So it's this commit https://github.com/pivpn/pivpn/commit/20d3a4ccd4f2d080d6f9a13a74e6369aa9d54476. There have been some non-criticial changes since then, see https://github.com/pivpn/pivpn/commits/master/, so if PiVPN works for you no need to do anything other than keeping the whole system up-to-date.

**Reply by ILudw1g** [reply]
ah okay thanks. appreciate that. i would still like to know though for future in case, 

with this commit, what is the best way to update the app if need be?

**Reply by orazioedoardo** [reply]
Probably won't be a way, see https://github.com/pivpn/pivpn/wiki/Upgrading
